packages/tfm-ror51-rubygem-sprockets-rails/tfm-ror51-rubygem-sprockets-rails.spec

```diff
@@ -0,0 +1,67 @@
+# Generated from sprockets-rails-3.2.1.gem by gem2rpm -*- rpm-spec -*-
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name sprockets-rails
+
+Name:    %{?scl_prefix}rubygem-%{gem_name}
+Version: 3.2.1
+Release: 1%{?dist}
+Summary: Sprockets Rails integration
+Group:   Development/Languages
+License: MIT
+URL:     https://github.com/rails/sprockets-rails
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildRequires: %{?scl_prefix_ruby}ruby >= 1.9.3
+BuildRequires: %{?scl_prefix}rubygem(railties) >= 4.0
```

Comment:
This can't be in the same PR since railties is added here.

Reply:
Is see you submitted https://github.com/theforeman/tfm-ror51-packaging/pull/14 for that.